### PR TITLE
Update Getting Started doc helm instructions

### PR DESCRIPTION
**Pull request by markfisher**

### CODE REVIEW

File: Getting-Started.adoc [hunk 1]
```diff
@@ -1,27 +1,27 @@
...
 ==== Prerequisites
 
-* A running 1.7 Kubernetes cluster with at least 4GB of memory. These instructions assume minikube for now.
+* A running 1.7+ Kubernetes cluster with at least 4GB of memory.
+
+For Minikube use:
 
 ----
-minikube start --cpus=4 --memory=4096 --disk-size=64g
+minikube start --cpus=4 --memory=4096
```
Comment (trisberg) [inline Getting-Started.adoc]: I would even remove the cpus setting, that has to vary by system

File: Getting-Started.adoc [hunk 2]
```diff
@@ -37,57 +37,57 @@ kubectl create clusterrolebinding tiller --clusterrole cluster-admin --serviceac
...
-helm repo update
+helm repo add riffrepo https://riff_charts.storage.googleapis.com
+helm repo update riffrepo
 ----
 
-==== Install Sk8s FaaS helm chart
+==== Install the riff helm chart
 
-Install the Sk8s Helm chart using one of the following commands.
+Install the riff helm chart as follows:
```
Comment (trisberg) [inline Getting-Started.adoc]: I think we should add a note saying that the user should run this from a checked out version of the `projectriff/riff` project. If not then the file references won't work.

Comment (trisberg) [reply Getting-Started.adoc]: Or, Maybe have a separate "Customizing the install" section where we reference the files in the `helm` directory

File: Getting-Started.adoc (hunk 1 above)
Comment (markfisher) [reply Getting-Started.adoc]: ok

File: Getting-Started.adoc (hunk 2 above)
Comment (markfisher) [reply Getting-Started.adoc]: I like the separate section idea

**Comment by markfisher** [comment]
@trisberg I addressed your comments

File: Getting-Started.adoc (hunk 2 above)
Comment (markfisher) [reply Getting-Started.adoc]: done

File: Getting-Started.adoc [hunk 3]
```diff
@@ -37,57 +37,68 @@ kubectl create clusterrolebinding tiller --clusterrole cluster-admin --serviceac
...
 
-For a cluster that does not have RBAC enabled use:
+- `helm search riffrepo/riff` and identify the version you're looking to install (use the latest one if unsure)
+
+For an install with default configuration and no RBAC enabled use:
 
 ----
-helm install --name demo sk8s-incubator/sk8s
+helm install riffrepo/riff --name demo --version <version>
```
Comment (trisberg) [inline Getting-Started.adoc]: don't think we need version, I've never seen anything but the latest get installed

File: Getting-Started.adoc [hunk 4]
```diff
@@ -37,57 +37,68 @@ kubectl create clusterrolebinding tiller --clusterrole cluster-admin --serviceac
...
 ----
 
-==== Install Sk8s FaaS helm chart
+==== Install the riff helm chart
 
-Install the Sk8s Helm chart using one of the following commands.
+Install the riff helm chart as follows:
 
-For a cluster that does not have RBAC enabled use:
+- `helm search riffrepo/riff` and identify the version you're looking to install (use the latest one if unsure)
```
Comment (trisberg) [inline Getting-Started.adoc]: there is no need to look up the version number, it should show the latest

File: Getting-Started.adoc (hunk 3 above)
Comment (markfisher) [reply Getting-Started.adoc]: sure

File: Getting-Started.adoc (hunk 4 above)
Comment (trisberg) [reply Getting-Started.adoc]: yes, I can do that on merge

**Comment by trisberg** [comment]
Merged as commit 3f75ab6686697f67e0a961c90460a70e4481d760

**Pull request closed without merging**